tests/google/cloud/sensors/test_bigquery.py

```diff
@@ -78,3 +78,15 @@ def test_redshift_sensor_async_execute_complete_event_none():
     )
     with pytest.raises(AirflowException):
         task.execute_complete(context=None, event=None)
+
+
+def test_poll_interval_deprecation_warning():
+    """Test DeprecationWarning for BatchSensorAsync by setting param poll_interval"""
```

Comment:
This is for BigQuery and not BatchSensory right? We need to correct the docstring.

Reply:
Fixed